PostGIS_Lidur3/Stadsetning.sql

```diff
@@ -0,0 +1,25 @@
+-- Finna allar staðsetningar sem tilheyra sjaldgæfustu staðsetningategundunum utan "The Seven Kingdoms"
+WITH type_counts AS (
+    -- Finna fjölda staðsetninga eftir staðsetningategund utan "The Seven Kingdoms"
+    SELECT l.type, COUNT(l.gid) AS location_count
+    FROM atlas.locations l
+    JOIN atlas.kingdoms k ON l.gid = k.gid
+    WHERE k.claimedby != 'The Seven Kingdoms'
```

Comment:
Ath. það er ekkert kingdom sem hefur `claimedby` með þetta gildi. Þannig þetta er ekki að útiloka réttu landsvæði. 

```
SELECT count(*) FROM atlas.kingdoms WHERE k.claimedby = 'The Seven Kingdoms'
```

```suggestion
    WHERE k.name not in  ('The North', 'The Vale', 'The Westerlands','The Riverlands', 'The Iron Islands','Dorne', 'The Stormlands', 'The Reach')
```